com.microsoft.mrtk.spatialmanipulation/ObjectManipulator/NewObjectManipulator.cs

```diff
@@ -0,0 +1,575 @@
+        /// as this restricts rotation of the object when we rotate the hand.
+        /// </summary>
+        public enum RotateAnchorType
+        {
+            RotateAboutObjectCenter,
+            RotateAboutGrabPoint
+        };
+
+        [System.Flags]
+        public enum ReleaseBehaviorType
+        {
+            KeepVelocity = 1 << 0,
+            KeepAngularVelocity = 1 << 1
+        }
+
+        #endregion Public Enums
+
+        #region Serialized Fields
+
+        [SerializeField]
+        [EnumFlags]
+        [Tooltip("What kinds of manipulation should be allowed?")]
+        private TransformFlags allowedManipulations = TransformFlags.Move | TransformFlags.Rotate | TransformFlags.Scale;
+
+        /// <summary>
+        /// What kinds of manipulation should be allowed?
+        /// </summary>
+        public TransformFlags AllowedManipulations
+        {
+            get => allowedManipulations;
+            set => allowedManipulations = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Rotation behavior of object when using one hand near")]
+        private RotateAnchorType rotationAnchorNear = RotateAnchorType.RotateAboutGrabPoint;
+
+        /// <summary>
+        /// Rotation behavior of object when using one hand near
+        /// </summary>
+        public RotateAnchorType RotationAnchorNear
+        {
+            get => rotationAnchorNear;
+            set => rotationAnchorNear = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Rotation behavior of object when using one hand at distance")]
+        private RotateAnchorType rotationAnchorFar = RotateAnchorType.RotateAboutGrabPoint;
+
+        /// <summary>
+        /// Rotation behavior of object when using one hand at distance
+        /// </summary>
+        public RotateAnchorType RotationAnchorFar
+        {
+            get => rotationAnchorFar;
+            set => rotationAnchorFar = value;
+        }
+
+        [SerializeField]
+        [EnumFlags]
+        [Tooltip("Rigid body behavior of the dragged object when releasing it.")]
+        private ReleaseBehaviorType releaseBehavior = ReleaseBehaviorType.KeepVelocity | ReleaseBehaviorType.KeepAngularVelocity;
+
+        /// <summary>
+        /// Rigid body behavior of the dragged object when releasing it.
+        /// </summary>
+        public ReleaseBehaviorType ReleaseBehavior
+        {
+            get => releaseBehavior;
+            set => releaseBehavior = value;
+        }
+
+        [FormerlySerializedAs("smoothingActive")]
+        [SerializeField]
+        [Tooltip("Frame-rate independent smoothing for far interactions. Far smoothing is enabled by default.")]
+        private bool smoothingFar = true;
+
+        /// <summary>
+        /// Whether to enable frame-rate independent smoothing for far interactions.
+        /// </summary>
+        /// <remarks>
+        /// Far smoothing is enabled by default.
+        /// </remarks>
+        public bool SmoothingFar
```

Comment:
I thought you said you moved Smoothing into the placement hub?

Reply:
I left these controls in here in this draft because the original object Manipulator interface allows for different smoothing behavior for far/near or near interactions. This wasn't quite collapsible back down to the placement hub (which is not an interactable), so these bools are surfaced here and control the placementhub in the background. 

The reasons for why it was separated out can be found here, basically, near smoothing didn't feel as good:
https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8204
https://github.com/MicrosoftDocs/mixed-reality/issues/291